app/views/organizations/index.html.haml

```diff
@@ -12,5 +12,7 @@
           .caption
             %h4
               = organization.name
-            %button.btn.btn-success Conferences
+            = link_to 'Conferences',
+              organization_conferences_path(organization),
```

Comment:
conferences_organization_path

Reply:
That's not a valid route.

```
> bundle exec rake routes
...
organization_conferences GET         /organizations/:organization_id/conferences(.:format)                                              organizations#conferences
....
```

Reply:
Never mind; I was on the wrong branch. I'll write a test to cover that. Nice catch!